src/src/tx_main.cpp

```diff
@@ -82,6 +83,9 @@ device_affinity_t ui_devices[] = {
 #ifdef HAS_BLE
   {&BLE_device, 0},
 #endif
+#ifdef HAS_BLET
+  {&BLET_device, 1},
```

Comment:
```suggestion
  {&BLET_device, 1},
```
What happens if you move to the "0" core in high-rate modes? Do you still observe packet rate jittering?

Reply:
It does not have any effect.

Packet rate 250, telemetry rate 1:2, phone connected - I see packet rate 250 in lua widget.
Packet rate 500:  when phone connects, I see ~450 packets in lua widget
Packet rate 1000: when phone connects, I see ~900 packets in lua widget

I think dropped packets are there due to FreeRTOS tasks created to maintain BLE connection, not from BLET device processing in ELRS. And we can not pin those to cores with Arduino framework.

It seems that those tasks can last up to 600us https://github.com/nkolban/esp32-snippets/issues/937
and are pinned to core 0 https://github.com/espressif/arduino-esp32/blob/master/tools/sdk/esp32/sdkconfig

So moving main loop to core 1 might help (only if there were easy way to do it).

Reply:
Currently, it's running on core 1 and it was my experiment in the original code if you look at the joystick define, it's running on core 0.

Reply:
As far I understand how nimble works is that you only update the values in memory and the closed blob does the updating, should not really cause the jitter unless the problem is at currently selected core.

Reply:
I tested witn  {&BLET_device, 0}, and {&BLET_device, 1} - there is no difference. What core should I put in the pool request?

Reply:
I think 0 same as `BLE_device`, maybe someone else choose 0 core for reason.